keyboards/mechanickeys/miniashen40/README.md

```diff
@@ -0,0 +1,26 @@
+# Mini Ashen 40
+
+![Mini Ashen 40](https://i.imgur.com/0k3gFZCh.jpeg)
+
+![Mini Ashen 40](https://i.imgur.com/B1w8qwPh.jpeg)
+
+![Mini Ashen 40](https://i.imgur.com/LqxXW0Rh.jpeg)
+
+A 40% keyboard with some extras
+  - Arrows
+  - Mini macro cluster
+  - Full assembly with only through hole components
+
+* Keyboard Maintainer: [jfescobar18](https://github.com/jfescobar18)
+* Hardware Supported: Mini Ashen 40 PCB and Case
+* Hardware Availability: [MechanicKeys](https://www.facebook.com/MechanicKeys-104963764775280)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```